lib/base-compiler.ts

```diff
@@ -288,6 +317,7 @@ export class BaseCompiler implements ICompiler {
             env.CC = this.compiler.exe;
         }
 
+        // todo: support changing of toolchainPath per compile
```

Comment:
can you maybe put some info in an issue and refer back to it here, like `TODO(#XYZW)`. I worry these TODOs get lost in the code unless they are an issue. Though in fairness they get lost there too 😊

Reply:
good point, almost forgot what this was about